client/src/Layout/Footer.tsx

```diff
@@ -0,0 +1,142 @@
+}))
+
+const FooterSrchIcon = styled(SendIcon)(({ theme }) => ({
+  color: "white",
+  "margin-right": "12px",
+}))
+
+
+export default function Footer() {
+  const { Support, Account } = footerData
+  return (
+    <>
+      <FooterBox>
+        <ConatinerFooter>
+          <Grid2 container columnSpacing={4}>
+            <Grid2 size={3}>
+              <Box>
+                <Footertext variant="h4">Exclisive</Footertext>
+                <Footertext variant="h6">Subscribe</Footertext>
+                <Footertext variant="subtitle2">
+                  Get 10% off your first order
+                </Footertext>
+                <FooterInput
+                  display={"flex"}
+                  justifyContent={"center"}
+                  alignItems={"center"}
+                >
+                  <Input
+                    type="email"
+                    label="Enter your email"
+                    name="emailfooter"
+                    variant="filled"
+                    styleProps={{
+                      "& .MuiInputLabel-filled": {
+                        color: "white",
+                      },
+                      "& .MuiFilledInput-underline:after": {
+                        border: 0,
+                      },
+                    }}
+                  ></Input>
+                  <FooterSrchIcon />
+                </FooterInput>
+              </Box>
+            </Grid2>
+
+            {Object.entries(footerData).map((value: [string, []], key: number) => {
+
+              return (
+                <>
+                  <Grid2 size={2}>
+                    <Box>
+                      <Footertext variant="h6">{value[0]}</Footertext>
+                      <Box>
+                        {value[1].map((value: string) => {
+
+                          return (
+                            <>
+                              <Footertext variant="subtitle2">
+                                {value}
+                              </Footertext>
+                            </>
+                          )
+                        })}
+                      </Box>
+                    </Box>
+                  </Grid2>
+                </>
+              )
+            })}
+
+            <Grid2 size={3}>
+              <Box>
+                <Footertext variant="h5">Download App</Footertext>
+                <Footertext variant="subtitle2">
+                  save $3 with App New User Only
+                </Footertext>
+
+                <Footimg src={Qrimagefooter} />
+
+                <Box display={"flex"} gap={3} color={"white"}>
+                  <FacebookIcon />
+                  <LinkedInIcon />
+                  <InstagramIcon />
+                  <TwitterIcon />
+                </Box>
+              </Box>
+            </Grid2>
+          </Grid2>
+        </ConatinerFooter>
+        <StyledDivider />
+        <Box textAlign={"center"}>
+          <Typography variant="subtitle2" sx={{ color: "gray", mt: 2 }}>
```

Comment:
you are repeating same mistake again and again  please  do not use inline style

Reply:
I am really sorry for that